code/game/gamemodes/miniantags/slaughter/bloodcrawl.dm

```diff
@@ -50,8 +48,9 @@
 					kidnapped = victim
 					stop_pulling()
 		flick("jaunt",animation)
-		loc = holder
-		holder = holder
+
+		src.holder = holder
```

Comment:
sometimes implied `src` is needed, when people do stupid things like this: `var/obj/effect/dummy/slaughter/holder = new /obj/effect/dummy/slaughter(mobloc)`